crates/driver/src/domain/competition/mod.rs

```diff
@@ -325,8 +325,7 @@ impl Competition {
 }
 
 /// Creates a vector with all possible combinations of the given solutions.
-/// The result is sorted by the number of merges, so the first elements are the
-/// original solutions.
+/// The result is sorted descending by score.
```

Comment:
The default `sort_by_key` sorts in ascending order, no?

Reply:
<img src="https://media2.giphy.com/media/2k5geABYZsReo/giphy.gif"/>